app/models/concerns/apple_delivery.rb

```diff
@@ -17,6 +17,10 @@ module AppleDelivery
     alias_method :apple_status, :apple_episode_delivery_status
   end
 
+  def publish_to_apple?
+    podcast.apple_config&.publish_to_apple?
```

Comment:
Ooh, I deleted the `podcast.apple_config` association in my PR.  I should re-add that as a memoized method or something.